packages/nextjs/test/integration/package.json

```diff
@@ -16,7 +16,8 @@
   "devDependencies": {
     "@types/node": "^15.3.1",
     "@types/puppeteer": "^5.4.3",
-    "@types/react": "^17.0.6",
+    // Hard-pinning @types/react because 17.0.48 causes build failures
+    "@types/react": "17.0.47",
     "@types/react-dom": "^17.0.5",
```

Comment:
We'll need hard pin `@types/react-dom` as well, they need to be kept in sync

Reply:
Pinned it to `17.0.17` which seems to be the last version (was released way before `@types/react@17.0.48`)

Reply:
Wait, how are you having comments in `package.json`??

Reply:
We're not, realized this as soon as I pushed it (fixed in [88c171c](https://github.com/getsentry/sentry-javascript/pull/5484/commits/88c171c61600225f473c56c7d4c8dd4f8d5079d6))

Reply:
Damn. I got all excited there for a minute...